tests/test_fock.py

```diff
@@ -75,14 +75,15 @@ def test_unsupported_gates(self):
         for g in all_gates - gates:
             op = getattr(qml.ops, g)
 
-            if op.num_wires == 0:
+            if op.num_wires <= 0:
```

Comment:
We might as well use the enums here. We made them for this purpose 😆

Reply:
Done!